packages/react-components/react-rating/library/src/components/Rating/useRating.tsx

```diff
@@ -11,6 +11,8 @@ import type { RatingProps, RatingState } from './Rating.types';
 import { RatingItem } from '../../RatingItem';
 import { StarFilled, StarRegular } from '@fluentui/react-icons';
 
+const defaultItemLabel = (num: number) => num + '';
+
```

Comment:
useRatingItem already has a default item label, we only need to pass the item label to the state so the user provided function is consumed.

```suggestion
```

Reply:
I was getting an exception when state was created with undefined itemLabel. I guess it was a temporary issue with my build. Applying both of the suggestions.